kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -105,7 +118,7 @@ local networkSpec = networkPolicy.mixin.spec;
                       "--model_name=" + name,
                       "--model_base_path=" + modelPath,
```

Comment:
soo...how will modelPath work with S3? That was what I meant on Slack - do you provide model_path like idk https://path-to-my-object? How does that work?

Reply:
s3://s3-bucket/model-path

Reply:
How does that work? I mean model-path is a directory. Moreover it's directory where you're supposed to have multiple versions of models - as in it'll look for dirs 1/<< model files>> 2/<< model files >> etc. Your approach will mean you'll have to have all the versions of models in same object, which is...not ideal.